src/matrices/skeletonization.jl

```diff
@@ -69,62 +69,29 @@ end
 
 # Transforms the standard RP into a close returns map
 function convert_recurrence_matrix(R::SparseMatrixCSC; triangle::Bool = true)
```

Comment:
The name `convert_recucrrence_matrix` is ambiguous. Since this is the counterpart of `revert_close_return_map`, I think that it would be better to reuse the name `create_close_returns_map` that is given to another function below. It would not be ambiguous, because the number of required arguments is different, so they would be two different methods.

Also, I have left `triangle` as a keyword argument to avoid breaking other parts of the code, but I see no strong reason to keep it as such. There are no more optional arguments, so it could be a "normal" positional argument.a keyword a

Reply:
Oh, I noticed that the "simplified" versions of the functions I have modified in this commit are not fully equivalent to the original ones written by @hkraemer : I have assumed that both the inputs and outputs are sparse _Boolean_ matrices, and that all nonzeros are `true`. On the other hand, the original implementation made "pseudo-rotations" of matrices with any type of elements.

That's very easy to fix: just take `nonzeros(R)` and pass its values as the third argument of `sparse` in the `return` lines, instead of a new vector made with `trues`. But I wonder if that's needed, or if my assumption was right and we can leave it as is.